index.html

```diff
@@ -18,7 +18,7 @@
     <div class="mobile-menu" id="id-for-js">
 
     </div>
```

Comment:
Great job @Nedramrez 🤝

- [ ] Kindly make sure the mobile popup options link to the intended references. When I click on about, I should be directed to the about section of the page which is not the case for your design. Kindly fix this issue and resubmit for approval.